### DISCUSSION TITLE
Incremental versions

### DISCUSSION THREAD

**Discussion by PJ-Zetier** [dvidelabs/flatcc#226] [Q&A]
Is there a plan to implement incremental versions.. the last tagged release was back in 2019.. It would be awesome if there was incremental tagging so that when including the repo in my source i dont have to refference a generic commit hash or rely on the old 0.6.0 version tag

**Comment by mikkelfj** [comment]
Yeah, it's been a bit slow with releases so things have piled up. I don't think there will be many incremental releases because not of new features will be added regularly, but the aim is to have one or two releases per year depending on what has been added. Currently a release is pending one pull request that needs to be refactored.

**Reply by PJ-Zetier** [reply]
Awesome! looking forward to the release and thanks for all the work on this!
